embedded-web-files/views/icons.html

```diff
@@ -0,0 +1 @@
+[[define "icon-plus-rectangle"]]<svg xmlns="http://www.w3.org/2000/svg" viewBox="-2 -2 24 24" width="24" height="24" preserveAspectRatio="xMinYMin" class="icon__icon"><path d="M10 20C4.477 20 0 15.523 0 10S4.477 0 10 0s10 4.477 10 10-4.477 10-10 10zm0-2a8 8 0 1 0 0-16 8 8 0 0 0 0 16zm1-7v4a1 1 0 0 1-2 0v-4H5a1 1 0 0 1 0-2h4V5a1 1 0 1 1 2 0v4h4a1 1 0 0 1 0 2h-4z"></path></svg>[[end]]
```

Comment:
:D it came with copy-paste from another project of mine. I kept it around in case I wanted a `plus` icon for a "New application" button. I didn't make it that far though. I kept the file around as a placeholder for icons.